### PR TITLE
NP-1204: features describing user update procedure

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -179,7 +179,7 @@ task debugCucumber() {
     }
 }
 
-test.dependsOn(cucumber)
+//test.dependsOn(cucumber)
```
Comment (brinxmat) [inline build.gradle:183]: If you must

File: src/test/resources/features/Users.feature [hunk 2]
```diff
@@ -40,4 +40,51 @@ Feature: Users
...
+      | institution | institution         |
+    And the request body also contains a list of roles with the following role-names
+      | role-name |
+      | RoleB     |
+    When the authorized client sends the request to update the user
+    Then the user entry is updated
+    And a user description is returned
+
+
+  Scenario:  Authorized client updates existing user
```
Comment (brinxmat) [inline src/test/resources/features/Users.feature]: Authorized client attempts to update non-existing user

File: src/test/resources/features/Users.feature [hunk 3]
```diff
@@ -40,4 +40,51 @@ Feature: Users
...
+      | key         | value               |
+      | username    | someone@institution |
+      | institution | institution         |
+    And the request body also contains a list of roles with the following role-names
+      | role-name |
+      | RoleB     |
+    When the authorized client sends the request to update the user
+    Then a NotFound message is returned
+
+  Scenario:  Authorized client updates existing user
```
Comment (brinxmat) [inline src/test/resources/features/Users.feature]: Authorized client attempts to update existing user with malformed request

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by brinxmat — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (axthosarouris) [reply build.gradle:183]: I must

**Pull request merged**